.eslintrc.json

```diff
@@ -1,7 +1,14 @@
 {
   "globals": {
-	"wp": "readable",
-	"jQuery": "readable"
+	  "wp": "readonly",
+	  "jQuery": "readonly",
```

Comment:
thanks for the changes @iamdharmesh, seems a quick fix of the vertical alignment is missed.